repository/src/test/java/org/pentaho/platform/repository/solution/filebased/SolutionRepositoryVfsFileObjectTest.java

```diff
@@ -1,13 +1,31 @@
 package org.pentaho.platform.repository.solution.filebased;
 
+import mockit.*;
+import mockit.integration.junit4.JMockit;
+import org.apache.commons.vfs2.FileContent;
+import org.apache.commons.vfs2.FileSystemException;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Ignore;
 import org.junit.Test;
-import org.pentaho.platform.api.repository2.unified.IUnifiedRepository;
-import org.pentaho.platform.api.repository2.unified.RepositoryFile;
-import org.pentaho.platform.api.repository2.unified.IAclNodeHelper;
-import org.pentaho.platform.api.repository2.unified.RepositoryFilePermission;
+import org.junit.runner.RunWith;
+import org.pentaho.platform.api.engine.IPentahoSession;
+import org.pentaho.platform.api.repository2.unified.*;
```

Comment:
Please recheck. This line is importing all the package. Maybe the next line is not needed